frontend/app/components/layouts/Footer.tsx

```diff
@@ -1,5 +1,43 @@
+'use client';
 import React from "react";
+import Image from "next/image";
 
 export default function Footer() {
-  return <div className=" bg-yellow-200 dark:bg-purple-400">Footer</div>;
-}
+
+  const handleLinkClick = () => {
+    // will open associated footer links, if any are actually implemented
+  };
```

Comment:
You don't need this if you use the Link component.
Make each link point to the homepage of twitter, youtube, instagram, etc...

```suggestion

```